km_api/know_me/tests/models/test_emergency_contact_model.py

```diff
@@ -0,0 +1,83 @@
+from know_me import models
+from django.core.exceptions import ValidationError
+from rest_framework.reverse import reverse
+import pytest
+
+
+def test_create(km_user_factory):
+    """
+    Test creating an emergency contact.
+    """
+    models.EmergencyContact.objects.create(
+        km_user=km_user_factory(),
+        name='First Last',
+        relation='Parent',
+        phone_number='19199199199',
+        alt_phone_number='12345678909',
+        email='hi@gmail.com')
+
+
+def test_get_absolute_url(emergency_contact_factory):
+    """
+    This method should return the URL of the emergency contact's
+    detail view.
+    """
+    ec = emergency_contact_factory()
+    expected = reverse(
+        'know-me:emergency-contact-detail',
+        kwargs={'pk': ec.pk}
+    )
+
+    assert ec.get_absolute_url() == expected
+
+
+def test_has_object_read_permission_owner(api_rf, emergency_contact_factory):
```

Comment:
We should also include the corresponding test for users who should not have read permissions on the contact. I believe this test is implemented for emergency items if you want a reference.